cypress/support/commands.js

```diff
@@ -19,6 +19,35 @@ Cypress.Commands.add("login", (email, password) => {
 });
 //
 
+
```

Comment:
Is this code still necessary to be left in the codebase?

Reply:
Removed old comments and code